docs/README.md

```diff
@@ -22,6 +22,18 @@ To run ImageLab from the command line:
   
  Note that on Windows platforms, the ":" character in the classpath must be changed to the ";" character.)
 
+ ### Run with BlueJ
+[Install BlueJ for your operating system](https://www.bluej.org/)
+* Clone the ImageLab repository to your local machine.
+* Run BlueJ
+* Select Project in the BlueJ menu > Open Project
+* Select the root folder of the cloned ImageLab repository.
+* Right click the box titled "Run" > Select "void main(String[] args)"
```

Comment:
Slight rewording of this instruction:

Right-click on the "Run" icon which pops up a menu. Select the "void main(String[] args)" method.